com.github.gabutakut.gabutdm.yml

```diff
@@ -0,0 +1,46 @@
+app-id: com.github.gabutakut.gabutdm
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+command: com.github.gabutakut.gabutdm
+finish-args:
+  - '--share=network'
+  - '--talk-name=com.canonical.Unity.LauncherEntry'
+  - '--share=ipc'
+  - '--socket=fallback-x11'
+  - '--socket=wayland'
+  - '--filesystem=xdg-download'
+cleanup:
+  - '*.a'
+  - '*.la'
+  - '/include'
+modules:
+  - name: libcanberra
+    buildsystem: autotools
+    sources:
+      - type: git
+        url: git://git.0pointer.net/libcanberra.git
+        tag: shared-modules
```

Comment:
this is not what I said at all.

Reply:
See https://github.com/flathub/flathub/wiki/App-Maintenance#required-files for some info